V2rayNG/app/src/main/kotlin/com/v2ray/ang/util/AngConfigManager.kt

```diff
@@ -318,7 +318,7 @@ object AngConfigManager {
                     result = Utils.decode(result)
                 }
 
-                val legacyPattern = "^(.+?):(.*)@(.+?):(\\d+?)$".toRegex()
+                val legacyPattern = "^(.+?):(.*)@(.+?):(\\d+?)/?$".toRegex()
```

Comment:
Thanks for your pr! Instead of `/?` I think adding `(\\/(.*)?)?` is better?
Since it also match an url with plugin parameter
`ss://cmM0LW1kNTpwYXNzd2Q=@192.168.100.1:8888/?plugin=obfs-local%3Bobfs%3Dhttp#Example2`

Reply:
Thanks for review!
But I have some questions.
Are plugins of shadowsocks in v2rayNG supported? I mean that if we support importing it, it will not work either.
And I guess that escape character `\` is not needed for `/`.
However, your proposal is harmless on the whole. Feel free to add your commits if you like to.

Reply:
If the regex is wrapped with `/`, it should escape, like PHP and ECMAScript.

Reply:
Tried in Android Studio, says it is redundant. Looks good. 👍